### DISCUSSION TITLE
Nushell

### DISCUSSION THREAD

**Discussion by uncomfyhalomacro** [catppuccin/catppuccin#2140] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

See https://www.nushell.sh/book/coloring_and_theming.html#coloring-and-theming-in-nu

### Any additional comments?

> Very nu. very shell. 🥺

I haven't started the port yet. I will probably do it after my fam member has recovered from the hospital in two weeks. 😘

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nullishamy** [comment]
I have some personal dots that do this, but would need refactored for use as a port.

**Reply by uncomfyhalomacro** [reply]
thats cool. are you willing to share your dots for the port? :3

**Reply by nullishamy** [reply]
https://github.com/nullishamy/derivation-station/blob/master/users/amy/apps/tools/nu/config.nu#L1

**Reply by uncomfyhalomacro** [reply]
@nullishamy thank you! i think your port is cool. i will just reference your username if i made the repository :) i tried it. it's so cool and pretty!

**Reply by Zinvoke** [reply]
I created a port a while ago just didn't make a separate repo:
https://github.com/nushell/nu_scripts/commit/c352f261548ee3c511bb4c0d3fa9a8bbeb8bba6c
You can find it here

**Reply by nullishamy** [reply]
Looks good. I think we should just reference this as part of our port. nu_scripts is a very standard place to get "extensions" for the shell. We can include instructions on how to install nu_scripts as part of our README, maybe provide some scripts to do it too.

**Comment by nullishamy** [comment]
Just circling back to this one, do we still want to make a port? How is the nu_scripts side of things getting on? Would love to hear more about what's gone on with this :eyes:

@uncomfyhalomacro @Zinvoke

**Comment by nik-rev** [comment]
I just completed the theme for Nushell. at https://github.com/catppuccin/catppuccin/issues/2641

**Comment by sgoudham** [comment]
Closing as further discussion should take place in #2641
